app/controllers/question_trackers_controller.rb

```diff
@@ -0,0 +1,79 @@
+# encoding: UTF-8
+
+class QuestionTrackersController < ApplicationController
+  before_filter :set_question_tracker, except: [:new, :index, :create]
+  before_filter :record_qt_origin
+
+  def index
+    authorize! :manage, QuestionTracker
+    @question_trackers = QuestionTracker.all # TODO Limit by user.
```

Comment:
I think we have to punt on these TODOs for now.  I think they should stay in. Maybe we could calculate technical debt by counting all the TODO tags in a source tree.